share/xtp/xml/dftgwbse.xml

```diff
@@ -1,81 +1,94 @@
+	                <gw_sc_max_iterations help="Maximum number of iterations in gvGW" default="50" choices="int+">50</gw_sc_max_iterations>
+	                <gw_mixing_order help="Mixing of QP energies in evGW - 0: plain, 1: linear, >1: Anderson" default="20" choices="int+">20</gw_mixing_order>
+                    <gw_sc_limit help="convergence criteria for shift" default="1e-5" choices="float+">1e-5</gw_sc_limit>
+	                <gw_mixing_alpha help="mixing alpha, also linear mixing" default="0.7" choices="float+">0.7</gw_mixing_alpha>
+                    <g_sc_max_iterations help="What is this again?" default="100" choices="int+">100</g_sc_max_iterations>
+                    <g_sc_limit help="convergence criteria for qp iteration (Hartree)" default="1e-5" choices="float+">1e-5</g_sc_limit>
+	                <use_Hqp_offdiag help="Using symmetrized off-diagonal elements of QP Hamiltonian in BSE" default="false" choices="bool">false</use_Hqp_offdiag>
+                    <sigma_plot_states help=""></sigma_plot_states>
+                    <sigma_plot_steps help="" default="201" choices="int+">201</sigma_plot_steps>
+                    <sigma_plot_spacing help="" default="1e-2" choices="float+">1e-2</sigma_plot_spacing>
+                    <sigma_plot_filename hep="File with the sigma plot data" default="QPenergies_sigma.dat">QPenergies_sigma.dat</sigma_plot_filename>
+                    <bse_print_weight help="print exciton WF composition weight larger than minimum" default="0.5" choices="float+">0.5</bse_print_weight>
+                    <rebuild_threecenter_freq help="how often the 3c integrals in iterate should be rebuilt" default="5" choices="int+">5</rebuild_threecenter_freq>
+                    <exctotal help="number of eigenvectors to calculate" default="5" choices="int+">5</exctotal>
+                    <ignore_corelevels help="" default="none" choices="none,RPA,BSE,GW">none</ignore_corelevels>
                 </gwbse>
             </gwbse_options>
-            <redirect_logger help="" default="0">0</redirect_logger>
+            <redirect_logger help="Redirect Logger" default="false" choices="bool">false</redirect_logger>
         </gwbse_engine>
 
         <geometry_optimization help="geometry optimization options">
             <state help="initial state to optimize for" default="s1">s1</state>
+            <maxiter help="Maximum iteration number" default="50" choices="int+">50</maxiter>
+            <trajectory_file help="File to write trajectory" default="optimisation.trj">optimisation.trj</trajectory_file>
             <statetracker help="tries to identify state without looking at the energy">
-                <oscillatorstrength default="0.0001">0.001</oscillatorstrength>
-                <filters help="property to track" default="oscillatorstrength">oscillatorstrength</filters>
+                <oscillatorstrength default="0.0001" choices="float+">0.001</oscillatorstrength>
```

Comment:
this is not correct anymore I think you can use a collection of filters now at the same time.

Reply:
@JensWehner